artemis-server/src/main/java/org/apache/activemq/artemis/core/server/plugin/ActiveMQServerPlugin.java

```diff
@@ -566,7 +566,7 @@ default void messageExpired(MessageReference message, SimpleString messageExpiry
     * @throws ActiveMQException
     */
    default void messageExpired(MessageReference message, SimpleString messageExpiryAddress, ServerConsumer consumer) throws ActiveMQException {
-
```

Comment:
these were part of a release... i wouldn't remove them... besides they don't bother anything.

Reply:
I'm calling the deprecated implementation here, I can't say why github show it as a deletion :open_mouth: